web/src/client/http.js

```diff
@@ -95,21 +95,21 @@ class HTTPClient {
 
   /**
    * @param {string} url - Endpoint URL (e.g., "/l10n/config").
-   * @return {Promise<object>} Server response.
+   * @return {Promise<Response>} Server response.
```

Comment:
Honest question, what is `Response`? https://developer.mozilla.org/en-US/docs/Web/API/Response, https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.response.html, or anything else?

Reply:
Nevermind, I see it is the first one, which is the what https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API returns.